### ISSUE TITLE
👌 IMPROVE: Add CI tests for jupyter notebooks

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-tutorials#115]
E.g. using something like https://github.com/computationalmodelling/nbval

**Comment by ltalirz** [comment]
@chrisjsewell I think this might be happening how (for the QB notebook)?

**Comment by chrisjsewell** [comment]
No not yet, but its certainly possible now using the docker environment I introduced in https://github.com/aiidateam/aiida-tutorials/pull/253.
We could either/or
1. just test that notebooks (well the solution variants) run successfully from start to end
2. Use nbval or my own https://pytest-notebook.readthedocs.io to diff a rerun notebook against a stored one (this entails storing the notebook with outputs in the repository, which is not currently done)
3. Use nteract's new package: https://github.com/nteract/testbook, which looks pretty interesting (I found out recently that nteract is owned by Netflix which is also pretty interesting: https://netflixtechblog.com/notebook-innovation-591ee3221233)

**Comment by ltalirz** [comment]
yeah, I saw that testbook on the scipy 2020 conference - already had a note written int he aiida meeting notes to ping you about it ;-)

**Comment by chrisjsewell** [comment]
Cheers, I noticed it because they use my myst-parser for their documentation 😉

**Comment by mbercx** [comment]
Now that we're moving to using MyST for all the tutorial content, we should into testing everything that we can be versus the latest `aiida-core` and `aiida-quantumespresso` releases in a nightly build, so we get pinged in case something breaks.
